### PR TITLE
Use openhab-webui features

### LINKED ISSUE #879: No webui's
Is it expected that webui's which have been split of recently cannot be installed anymore in the snapshots?

CC: @davidgraeff

### PR DESCRIPTION
**Pull request by wborn**
* Adds features from openhab-webui
* Removes UI features which are added by https://github.com/openhab/openhab-webui/pull/9

Fixes #879

### CODE REVIEW

**Comment by cweitkamp** [comment]
Will this PR fix our failing Travis build too? Or do we have to change a dependency in the OH2 Add-ons repository to solve it?

```
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: org.openhab.io.rest.docs 2.5.0.qualifier
[ERROR]   Missing requirement: org.openhab.io.rest.docs 2.5.0.qualifier requires 'java.package; org.openhab.ui.dashboard 0.0.0' but it could not be found
[ERROR] 
[ERROR] See http://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
[ERROR] Cannot resolve dependencies of MavenProject: org.openhab.io:org.openhab.io.rest.docs:2.5.0-SNAPSHOT @ /home/travis/build/openhab/openhab2-addons/addons/io/org.openhab.io.rest.docs/pom.xml: See log for details -> [Help 1]
```

**Comment by wborn** [comment]
I think the openhab2-addons build needs another fix. Haven't looked at it but perhaps the root POM may need an additional repository defined where it can download the UI artifacts from.

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
         <online.repo.version>${parsedVersion.majorVersion}.${parsedVersion.minorVersion}</online.repo.version>        
 
         <ohc.version>2.5.0-SNAPSHOT</ohc.version>
+        <ohw.version>2.5.0-SNAPSHOT</ohw.version>
```
Comment (kaikreuzer) [inline pom.xml]: I don't think we need to add a specific version - using `project.version` should be just fine (that's what we do for other addons as well).

File: distributions/openhab/src/main/filtered-resources/userdata/etc/version.properties [hunk 2]
```diff
@@ -7,6 +7,7 @@ Repository        Version
 ----------------------------------------
 openhab-distro  : ${project.version}
 openhab-core    : ${ohc.version}
+openhab-webui   : ${ohw.version}
```
Comment (kaikreuzer) [inline distributions/openhab/src/main/filtered-resources/userdata/etc/version.properties]: remove this one (UIs are still considered a part of openhab2-addons, just as zigbee and zwave bindings)

**Comment by kaikreuzer** [comment]
Thanks for dealing with that @wborn! This is indeed a missing step, which I planned to go for once the "move UI" PRs were merged, but then I fell sick and I am spending my time in bed atm 😢.

I think @cweitkamp is right and we still have a problem with the io.rest.docs as this has a dependency to the dashboard as well. We might want to move that to webui as well as @davidgraeff suggested (some where).

**Comment by wborn** [comment]
I'm fine with reusing `project.version` as well so it should be fixed now. 
Hope you get well soon too @kaikreuzer!

**Comment by lucacalcaterra** [comment]
I'm working on a binding development, today i create a fresh copy of the environment but launching openhab runtime hangs to please stand by while ui are being installed.

Is this the issue ? But i didn't understand how to fix it..

File: launch/Jenkinsfile [hunk 3]
```diff
@@ -174,7 +173,6 @@ def releaseOpenHabComponent(componentName, branch, releaseVersion, nextVersion,
                             sh "mvn versions:update-parent -DallowSnapshots=true -DparentVersion=[" + env.OH2_NEXT_VERSION +"-SNAPSHOT] " + mvnOptions
                         }
                         if(updateProperties) {
-                            sh "mvn versions:set-property -Dproperty=esh.version -DnewVersion="+ env.ESH_NEXT_VERSION +"-SNAPSHOT " + mvnOptions
```
Comment (martinvw) [inline launch/Jenkinsfile]: @wborn, would it be related if I now get this error:

```
openhab> feature:install esh.tp-commons-net
Error executing command: No matching features for esh.tp-commons-net/0
```

You suggested installing that https://github.com/openhab/openhab2-addons/issues/4709#issuecomment-461855772

Comment (wborn) [reply launch/Jenkinsfile]: It's unrelated because this line was just for updating the (no longer existing) `esh.version` property in case of a release.

**Pull request merged**